domain/schema/model/sql/0020-unit.sql

```diff
@@ -52,6 +52,17 @@ ON unit (application_uuid);
 CREATE INDEX idx_unit_net_node
 ON unit (net_node_uuid);
 
+CREATE TABLE unit_channel (
```

Comment:
We're also missing the os. Take a look at `application_platform`.

Reply:
As discussed online, I also added OS and arch.